spec/models/search_spec.rb

```diff
@@ -0,0 +1,272 @@
+require "spec_helper"
+
+describe Search do
+
+  let(:setup_data) do
+    create(:node, label: "test")
+    create(:note, text: "test")
+    create(:issue, text: "test")
+    create(:evidence, content: "test")
+  end
+
+  describe "#all" do
+
+    it "filters nodes, notes, issues, evidence by matching search term" do
+      setup_data
+      q = "test"
+
+      results = Search.new(query: q, scope: :all).results
+      expect(results.size).to eq 4
+    end
+
+    it "shows count based on matched fields" do
+      setup_data
+      q = "test"
+
+      search = Search.new(query: q, scope: :all)
+
+      expect(search.total_count).to eq 4
+      expect(search.notes_count).to eq 1
+      expect(search.issues_count).to eq 1
+      expect(search.nodes_count).to eq 1
+      expect(search.evidence_count).to eq 1
+    end
+  end
+
+  describe "#total_count" do
+    it "returns 0 if noting founded" do
```

Comment:
typo: "noting" for "nothing"; the same mistake is repeated a few other times in this file.